src/Symfony/Component/Serializer/Normalizer/GetSetMethodNormalizer.php

```diff
@@ -97,7 +97,7 @@ private function supports(string $class): bool
     private function isGetMethod(\ReflectionMethod $method): bool
     {
         return !$method->isStatic()
-            && !$method->getAttributes(Ignore::class)
+            && !$method->getAttributes(Ignore::class, \ReflectionAttribute::IS_INSTANCEOF)
```

Comment:
I think it'd be better to check the LegacyIgnore attribute directly instead of opening the attribute to inheritance (which opens to a potentially infinite number of child attributes)

Reply:
fixed